src/components/cell-menu-container.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react'
+import { connect } from 'react-redux'
+import { bindActionCreators } from 'redux'
+import PropTypes from 'prop-types'
+import Tooltip from 'material-ui/Tooltip'
+
+// import CellRow from './cell-row'
+// import { CellContainer } from './cell-container'
+
+// import CellEditor from './cell-editor'
+
+import * as actions from '../actions'
+import { getCellById } from '../notebook-utils'
+
+
+export class CellMenuContainerUnconnected extends React.Component {
+  static propTypes = {
+    label: PropTypes.string.isRequired,
+  }
+
+  render() {
+    return (
+      <div className="cell-menu-container">
+        <Tooltip
+          classes={{ tooltip: 'iodide-tooltip' }}
+          placement="bottom"
+          title="Cell Settings"
+          enterDelay={600}
+        >
+          <div className="cell-type-label">{this.props.label}</div>
+        </Tooltip>
+      </div>
+    )
+  }
+}
+
+
+function mapStateToProps(state, ownProps) {
+  const { cellId } = ownProps
+  const cell = getCellById(state.cells, cellId)
+  let label
+  if (cell.cellType === 'code') {
+    label = cell.language
+  } else if (cell.cellType === 'markdown') {
+    label = 'md'
+  } else if (cell.cellType === 'css') {
+    label = 'css'
+  } else if (cell.cellType === 'plugin') {
+    label = 'plugin'
+  } else if (cell.cellType === 'external dependencies') {
+    label = 'resource'
```

Comment:
I notice that cells with "respource" overflow a bit. I bet we could do one of these two things:

1.) change `resource` to `res` - we are already using md, js, etc. and maybe `ext` or `dep` clear.
2.) change the font size defined for `cell-type-label` to `12px`. This is still legible, and everything eems to fit ok.